### PR TITLE
[azopenai] Make sure all the examples are runnable.

### PR DESCRIPTION
**Pull request by richardpark-msft**
I consolidated all environment variables in a previous PR and now the examples can use them as well (with nicer aliases).

### CODE REVIEW

File: sdk/ai/azopenai/go.mod
Comment (jhendrixMSFT) [inline sdk/ai/azopenai/go.mod:1]: Can you please do a `go get -u all` followed by `go mod tidy` to get this current?

**Review by richardpark-msft — COMMENTED** [review]
/azp run go - azopenai

**Review by richardpark-msft — COMMENTED** [review]
/azp run go - azopenai

**Comment by richardpark-msft** [comment]
(manually disabled old revisions of secrets in the keyvault for this pipeline)

**Pull request merged**